### PR TITLE
Minor UX improvements

### PR DESCRIPTION
**Pull request by bergie**
* [x] Fix touchscreen pinch-zoom
* [ ] Make node dragging on mouse more robust
* [ ] Re-enable creating edges by dragging port

### CODE REVIEW

**Comment by bergie** [comment]
@jonnor I think we should probably merge these before continuining with #364

File: the-graph/the-graph-app.js [hunk 1]
```diff
@@ -159,8 +159,8 @@ module.exports.register = function (context) {
         scale: fit.scale,
         width: this.props.width,
         height: this.props.height,
-        minZoom: this.props.minZoom,
-        maxZoom: this.props.maxZoom,
+        minZoom: this.props.minZoom || 0.15,
```
Comment (jonnor) [inline the-graph/the-graph-app.js]: Or alternatively that an invalid value for props.minZoom is getting in from some other source.

Comment (bergie) [inline the-graph/the-graph-app.js]: Yeah, I was seeing `NaN` there

**Comment by jonnor** [comment]
Reverted that one chance, it does not seem to be neccesary now that #365 is in

**Pull request merged**